### PR TITLE
manylinux CI

### PR DESCRIPTION
**Pull request by darioizzo**
Adding the manylinux packages creation to the current CI

### CODE REVIEW

File: tools/travis_manylinux.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+export deps_dir=$HOME/local
+export PATH="$HOME/miniconda/bin:$PATH"
+export PATH="$deps_dir/bin:$PATH"
+
+docker pull ${DOCKER_IMAGE};
+docker run --rm -e TWINE_PASSWORD -e PYGMO_BUILD_TYPE -e TRAVIS_TAG -v `pwd`:/pygmo2 $DOCKER_IMAGE bash /pygmo2/tools/docker_manylinux.sh
+
+set +e
+set +x
```
Comment (bluescarni) [inline tools/travis_manylinux.sh]: Missing newline here.

File: tools/wheel_setup.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import os
+from setuptools import setup
+from setuptools.dist import Distribution
+import sys
+
+NAME = 'pygmo'
+VERSION = '@pygmo_VERSION@'
+DESCRIPTION = 'Parallel optimisation for C++ / Python'
```
Comment (bluescarni) [inline tools/wheel_setup.py]: Probably drop the "C++" here.

File: tools/wheel_setup.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+LONG_DESCRIPTION = 'A platform to perform parallel computations of optimisation tasks (global and local) via the asynchronous generalized island model.'
+URL = 'https://github.com/esa/pagmo2'
```
Comment (bluescarni) [inline tools/wheel_setup.py]: URL needs to be updated.

File: tools/wheel_setup.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+AUTHOR = 'The pagmo development team'
+AUTHOR_EMAIL = 'pagmo@googlegroups.com'
+LICENSE = 'GPLv3+/LGPL3+'
```
Comment (bluescarni) [inline tools/wheel_setup.py]: The license for pygmo has changed, now it is ``'MPL-2.0'`` (i.e., Mozilla Public License 2.0).

File: tools/wheel_setup.py [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    'Development Status :: 4 - Beta',
+
+    'Operating System :: OS Independent',
+
+    'Intended Audience :: Science/Research',
+    'Topic :: Scientific/Engineering',
+    'Topic :: Scientific/Engineering :: Mathematics',
+    'Topic :: Scientific/Engineering :: Physics',
+
+    'License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)',
```
Comment (bluescarni) [inline tools/wheel_setup.py]: Derop these two lines with the licenses, we need only one line containing:

```
'License :: OSI Approved :: Mozilla Public License 2.0 (MPL-2.0)',
```

File: tools/travis_manylinux.sh (hunk 1 above)
Comment (darioizzo) [reply tools/travis_manylinux.sh]: how did you catch this!!

Comment (bluescarni) [reply tools/travis_manylinux.sh]: https://media.giphy.com/media/12NUbkX6p4xOO4/giphy.gif

**Pull request merged**